### ISSUE TITLE
paper-ripple focus state issue

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-ripple#13]
The focus state is not visible. We need a clearer way to identify which item is in focus.

**Comment by notwaldorf** [comment]
I think this has been fixed (the demo elements have focus outlines for everything)
